language/analysis/src/main/java/com/google/cloud/language/samples/Analyze.java

```diff
@@ -136,7 +138,7 @@ public static void printSyntax(PrintStream out, List<Token> tokens) {
   /**
```

Comment:
@gguuss The analyzeSentiment is still returning Mag and Polarity. It should return score and mag.